app/src/main/java/io/neurolab/activities/MeditationActivity.java

```diff
@@ -101,6 +89,20 @@ private void initializePlaybackController() {
         Log.d(TAG, "Inside initializePlaybackController: MediaPlayerHolder progress callback set");
     }
 
+    private String getTimeString(long millis) {
+        StringBuffer buf = new StringBuffer();
+
+        int minutes = (int) (millis % (1000 * 60 * 60)) / (1000 * 60);
+        int seconds = (int) ((millis % (1000 * 60 * 60)) % (1000 * 60)) / 1000;
+
+        buf
```

Comment:
Reformat this by removing unnecessary spacing and indents.

Reply:
Change made :+1: